V0 Skirt Collection

harry-boe opened the pull request:
This is a mod with different Skirts as described in the readme.
I followed the conventions below

Harry


<!--
Thank you for your interest in contributing to the VoronUsers repository, it is
highly appreciated!

**Please make sure the submission conforms to the rules outlined below. PRs which fail
to conform to the rules below are likely to be rejected.**
-->

  * [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folder and file naming:
    * Folders and filenames shouldn't contain spaces. Only letters `a-zA-Z`, numbers `0-9`, underscores `_`, hyphens `-` and periods `.`
    * Primary color: `part_xyz.stl`
    * Accent color: `[a]_part_xyz.stl`
    * Opaque color (Blocks light): `[o]_part_xyz.stl`
    * Clear/transparent color (Allows light): `[c]_part_xyz.stl`
    * Quantity, if more than one is needed: `part_xyz_x4.stl`
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

<!--
Describe the submission further using the template provided below. The more 
details the better!
-->

#### Which mods/configurations/profiles are added by this PR?

#### How was it tested? 

#### Any background context you want to provide?

#### Screenshots (if appropriate)

#### Further notes

MapleLeafMakers commented on printer_mods/README.md:
```diff
@@ -186,6 +186,7 @@ like so:
 | | [Voron2.4 Y Endstop Relocation](./hartk1213/Voron2.4_Y_Endstop_Relocation) | This mod relocates the Y endstop to the top A drive piece | :x:| :x: | :heavy_check_mark: |:x:|:x: |
 | | [Voron2.4 GE5C](./hartk1213/Voron2.4_GE5C) | This mod replaces the stack of printed z joints with a metal spherical bearing  | :x:| :x: | :heavy_check_mark: |:x:|:x:|
 | | [Voron2.4 SexBolt ZEndstop](./hartk1213/Voron2.4_SexBolt_ZEndstop) | This mod replaces stock V2 Z endstop with a different style| :x:| :x: | :heavy_check_mark: |:x:|:heavy_check_mark: |
+| hbo | [Voron0 skirs](./hbo/Voron0_Skirts) | Collection of Voron 0 Skirts | :heavy_check_mark:| :x: | :x: |:x:|
```
typo in `skirs`

harry-boe replied:
fixed

MapleLeafMakers commented on printer_mods/hbo/Voron0_Skirts/Readme.md:
```diff
@@ -0,0 +1,86 @@
+
+![Printed Panel](Images/VORON_V0.1_2022.png)
+
+Right now there are 3 variations for the and one commmon rear.
```
"3 variations for the...." I think you missed a word here.

harry-boe replied:
fixed

MapleLeafMakers commented on printer_mods/hbo/Voron0_Skirts/Readme.md:
```diff
@@ -0,0 +1,86 @@
+The rear skirts match all the different front skirts and have the following features
+
+- integrated power inlet mount. To be used with the "standard" v0 power inlet
+- 2 keystone mount to get acces to the raspery ports
```
`acces` -> `access`
`raspery` -> `raspberry pi`

harry-boe replied:
fixed

MapleLeafMakers commented on printer_mods/hbo/Voron0_Skirts/Readme.md:
```diff
@@ -0,0 +1,86 @@
+
+- integrated power inlet mount. To be used with the "standard" v0 power inlet
+- 2 keystone mount to get acces to the raspery ports
+- bowden clip mount support. Disigned for the E3D style bowden clips. This allows to push the fillament and not the tube from the outside. Read for the rabit feader :)
```
`Disigned` -> `Designed`
`fillament` -> `filament`
`rabit feader` -> `rabbit feeder`  (Consider calling this by it's full name "Enraged Rabbit Carrot Feeder", or just ERCF.

harry-boe replied:
fixed

MapleLeafMakers commented on printer_mods/hbo/Voron0_Skirts/Readme.md:
```diff
@@ -0,0 +1,86 @@
+
+## Headless
+
+This version is intendet for users that like to run the V0 in headless mode i.e. without display
```
`intendet` -> `intended`

harry-boe replied:
fixed

MapleLeafMakers commented on printer_mods/hbo/Voron0_Skirts/Readme.md:
```diff
@@ -0,0 +1,86 @@
+
+## Mini Display
+
+This version is desiged for the 12864 mini display (same as the V2.4 display)
```
`12864 mini` -> `mini 12864`

MapleLeafMakers commented on printer_mods/hbo/Voron0_Skirts/Readme.md:
```diff
@@ -0,0 +1,86 @@
+
+## Waveshare 4.3" Display
+
+This version is desiged for the Waveshare 4.3inch Capacitive Touch Display for Raspberry Pi, DSI Interface
```
`desiged` -> `designed`

MapleLeafMakers commented on printer_mods/hbo/Voron0_Skirts/Readme.md:
```diff
@@ -0,0 +1,86 @@
+## Mini Display
+
+This version is desiged for the 12864 mini display (same as the V2.4 display)
+The design includes a little part to get acces to the reset switsch.
```
`acces` -> `access`
`switsch` -> `switch`

MapleLeafMakers commented on printer_mods/hbo/Voron0_Skirts/Readme.md:
```diff
@@ -0,0 +1,86 @@
+## Waveshare 4.3" Display
+
+This version is desiged for the Waveshare 4.3inch Capacitive Touch Display for Raspberry Pi, DSI Interface
+The mounts are desihed to bolt on the frame and display with original screws from the back.
```
`desihed` -> `designed`

MapleLeafMakers commented on printer_mods/hbo/Voron0_Skirts/Readme.md:
```diff
@@ -0,0 +1,86 @@
+This version is desiged for the Waveshare 4.3inch Capacitive Touch Display for Raspberry Pi, DSI Interface
+The mounts are desihed to bolt on the frame and display with original screws from the back.
+Therfore similar 4.3 inch display might not work without modifications
+I'm using a 60cm DSI cable to connect the diplay with a Raspery back on the back of the V0 
```
`Raspery` -> `Raspberry Pi`

MapleLeafMakers commented on printer_mods/hbo/Voron0_Skirts/Readme.md:
```diff
@@ -0,0 +1,86 @@
+(a Raspberry Pico mount mod is in preparation)
+
+### Reset / Emergency switch
+The fron display mount also provide a mounting option for a reset or inmergency switch.
```
`fron` -> `front`
`inmergency` -> `emergency`

MapleLeafMakers commented on printer_mods/hbo/Voron0_Skirts/Readme.md:
```diff
@@ -0,0 +1,86 @@
+The fron display mount also provide a mounting option for a reset or inmergency switch.
+I'm using this switch to reset a BTT PSU relays. 
+
+The switch in the cad is a standard 7mm momentray switch that is easy to source even from local providers.
```
`momentray` -> `momentary`

MapleLeafMakers commented on printer_mods/hbo/Voron0_Skirts/Readme.md:
```diff
@@ -0,0 +1,86 @@
+I'm using this switch to reset a BTT PSU relays. 
+
+The switch in the cad is a standard 7mm momentray switch that is easy to source even from local providers.
+Make shire to get a Normal OPEN switch if you plan to use it with the BTT PSU Relays.
```
`shire` -> `sure`
`Normal OPEN` -> `Normally Open (NO)`

MapleLeafMakers commented on printer_mods/hbo/Voron0_Skirts/Readme.md:
```diff
@@ -0,0 +1,86 @@
+
+All parts print with standard Voron print setting without any supports.
+All parts fit on a V0. However the max out the build volume on the z axis.
+If you need to print on a V0 make shure that you bed springs are really stiff by adding more tention with washers etc.
```
`shure` -> `sure`
`you` -> `your`
`tention` -> `tension`

harry-boe replied:
fixed

MapleLeafMakers commented on printer_mods/hbo/Voron0_Skirts/Readme.md:
```diff
@@ -0,0 +1,86 @@
+All parts print with standard Voron print setting without any supports.
+All parts fit on a V0. However the max out the build volume on the z axis.
+If you need to print on a V0 make shure that you bed springs are really stiff by adding more tention with washers etc.
+I had some problems getting nice parts out of the V0 due to the nozzle mooving he part dur to flex on the bed springs.
```
`mooving` -> `moving`
`he` -> `the`
`dur` -> `due`

harry-boe replied:
fixed

MapleLeafMakers commented on printer_mods/hbo/Voron0_Skirts/Readme.md:
```diff
@@ -0,0 +1,86 @@
+
+12864 Mini Display:  https://biqu.equipment/collections/lcd/products/bigtreetech-mini12864-v1-0-lcd-screen-rgb-backlight-mini-display-supports-marlin-diy-for-skr-3d-printer-part
+
+Rubber Feets: Ther are many options. The one in the CAD are this ones here https://de.aliexpress.com/item/1005004478863418.html?spm=a2g0o.order_list.0.0.13575c5f8HKvPU&gatewayAdapt=glo2deu
```
`Feets` -> `Feet`
`Ther` -> `There`
`this` -> `these`

harry-boe replied:
fixed

MapleLeafMakers commented on printer_mods/hbo/Voron0_Skirts/Readme.md:
```diff
@@ -0,0 +1,86 @@
+
+## CAD  Notes
+
+The CAD includes all varients in one file.
```
`varients` -> `variants`

harry-boe replied:
fixed

MapleLeafMakers commented on printer_mods/hbo/Voron0_Skirts/Readme.md:
```diff
@@ -0,0 +1,86 @@
+## CAD  Notes
+
+The CAD includes all varients in one file.
+You need to show/hode the version you like to work with
```
`hode` -> `hide`

harry-boe replied:
fixed

MapleLeafMakers reviewed (commented):
- File and folder cannot contain spaces
   - printer_mods/hbo/Voron0_Skirts/CAD/AZ Skirts VORON_V0.1.f3z
   - printer_mods/hbo/Voron0_Skirts/CAD/AZ Skirts VORON_V0.1.step.zip

harry-boe commented:
As in the commit description.

Removed spaces in  CAD filenames
Typos Fixed in readme's
Cleanup of links to Aliexpress sources (removed not required infos in the url params)
+ Added description for skirt fans
+ Added missing Bottom Plate stl file
+ added file name prefix for agent colors

andrepl commented:
The changes look good, but you've missed a few of the typos in the readme, they're all the ones hidden behind this: 
![image](https://user-images.githubusercontent.com/382747/197427001-d5a63d70-5361-4e0a-8357-ebfea1795442.png)

harry-boe commented:
readme updated and checked with a spell checker

MapleLeafMakers commented:
just 1 left:
![image](https://user-images.githubusercontent.com/105010153/197539258-dc8df0da-4fb4-4f85-8658-f9a928fba584.png)

harry-boe commented:
last typos fixed

harry-boe commented:
Is there anything I need to do to get this merged ?

harry-boe commented:
it's been a while now. Anything I need to do here ?

FHeilmann commented:
Hi, 

I want to thank you for your submission and want to personally apologize that your submission has not gotten the review it deserves.
We are in the process to restructure the repo to streamline reviews and hopefully solve this issue in the future.

Unfortunately, this means that we will require you to resubmit the mod under the new system.

Please check [here](https://github.com/VoronDesign/VoronUsers/wiki/VoronUsers-Overhaul) for details and instructions on how to proceed. 
If you have any questions, please reach out in the #voronuser_mods channel on the VoronDesign discord.
If you have since moved on from this mod and don't want to resubmit, no further action is necessary from your side.

Bests,

Florian a.k.a. Haribro
